docs/next.config.js

```diff
@@ -105,6 +105,7 @@ module.exports = withDocsInfra({
       },
     };
   },
+  distDir: 'export',
   // Next.js provides a `defaultPathMap` argument, we could simplify the logic.
   // However, we don't in order to prevent any regression in the `findPages()` method.
   exportPathMap: () => {
```

Comment:
Next.js deprecating this https://nextjs.org/docs/app/api-reference/next-config-js/exportPathMap worries me. How are we supposed to have pages under /api/ once it's gone? But I guess that now since we have product folded prefixes in the URLs, we don't need to worry about the conflict with the Next.js backend API route. We can rename `api-docs` -> `api` everywhere.